docs/core/porting/upgrade-assistant-uwp-framework.md

```diff
@@ -0,0 +1,524 @@
+---
+title: Upgrade UWP apps to .NET 6/WinUI3/Windows App SDK
+description: Use the .NET Upgrade Assistant to upgrade an existing UWP app to .NET 6. The .NET Upgrade Assistant is a CLI tool that helps migrating an app from .NET Framework to .NET 6.
+author: adegeo
+ms.date: 05/17/2022
+---
+# Upgrade a UWP App to .NET 6 and WinUI3 with Windows App SDK capabilities using the .NET Upgrade Assistant
+
+The [.NET Upgrade Assistant](upgrade-assistant-overview.md) is a command-line tool that can assist with upgrading .NET Framework WPF apps to .NET 6. This article provides:
+
+- What to expect from the tool?
+- Things to know before starting your migration journey 
+- A demonstration of how to run the tool against a UWP app
+- Troubleshooting tips
+
+For more information on how to install the tool, see [Overview of the .NET Upgrade Assistant](upgrade-assistant-overview.md).
+
+## What to expect from the tool?
+
+We’re working on an update to the .NET Upgrade Assistant which is an open-source tool to help you automate migration of .NET Native UWP apps to .NET 6, including updating from WinUI 2 to WinUI 3. This is a command line tool that leverages .NET Upgrade assistant and will live as an extension in the same github repo [here](https://github.com/dotnet/upgrade-assistant.git). This tool aims to provide a simple migration experience by 
+
+- Backing up your UWP project 
+- Convert project file (csproj) to SDK style
+- Clean up NuGet package references 
+- Update the target framework and NuGet packages 
+- Update package.appxmanifest
+- Removal of properties and items that are no longer required
+- Add template files such as App.Xaml, MainWindow.Xaml and publish profiles with options to build packaged and unpackaged app
+- Make namespace changes and add MainPage navigation
+- An attempt to detect and fix apis that have changed from UWP to Windows App SDK
+- An attempt to detect and mark apis that are either no longer supported or cannot be automatically converted with TODO comments and CLI messages
```

Comment:
Considering the amount of items and how long these are, you should use a period at the end of each line. Also, can you tighten this up? The last two lines are pretty similar, other items don't really say much like "update package.appxmanifest" doesn't really mean anything, all sorts of files will be updated.

What about
```suggestion
- Back up your project.
- Converts your project to the latest SDK format and cleans up your NuGet package references.
- Adds new template files such as _App.Xaml_, _MainWindow.Xaml_ and publish profiles.
- Updates namespaces and adds **MainPage** navigation.
- Attempts to detect and fix APIs that have changed, and marks APIs that are no longer supported, with `//TODO` code comments.
```

Reply:
Your input looks good.